### ISSUE TITLE
Iteration steps needed for SSD_Mobilenet_V1_coco

### ISSUE THREAD

**Issue by RajMalar** [EdjeElectronics/TensorFlow-Object-Detection-API-Tutorial-Train-Multiple-Objects-Windows-10#22]
I am using SSD_Mobilenet_V1_coco for training.... the loss is below 1 (no of iterations: 18695) but the model can't able to detect any objects.....what is required loss rate to detect the objects?..
